processor/metricstransformprocessor/metrics_transform_processor_testcases_test.go

```diff
@@ -411,18 +473,25 @@ var (
 				},
 			},
 			in: []*metricspb.Metric{
-				metricBuilder().setName("metric1").setLabels([]string{"label1", "label2"}).setDataType(metricspb.MetricDescriptor_GAUGE_INT64).
-					addTimeseries(3, []string{"label1-value1", "label2-value1"}).addInt64Point(0, 3, 2).
-					addTimeseries(0, []string{"label1-value1", "label2-value4"}).
-					addTimeseries(3, []string{"label1-value1", "label2-value2"}).addInt64Point(2, 1, 2).addInt64Point(2, 2, 3).
-					addTimeseries(1, []string{"label1-value1", "label2-value3"}).addInt64Point(3, 1, 2).
+				metricBuilder().setName("metric1").setLabels([]string{"label1", "label2"}).
+					setDataType(metricspb.MetricDescriptor_GAUGE_INT64).
+					addTimeseries(3, []string{"label1-value1", "label2-value1"}).
+					addInt64Point(0, 3, 2).
+					addTimeseries(3, []string{"label1-value1", "label2-value2"}).
+					addInt64Point(1, 1, 2).addInt64Point(1, 2, 3).
+					addTimeseries(1, []string{"label1-value1", "label2-value3"}).
+					addInt64Point(2, 1, 2).
 					build(),
 			},
 			out: []*metricspb.Metric{
-				metricBuilder().setName("metric1").setLabels([]string{"label1", "label2"}).setDataType(metricspb.MetricDescriptor_GAUGE_INT64).
-					addTimeseries(1, []string{"label1-value1", "label2-value3"}).addInt64Point(0, 1, 2).
-					addTimeseries(3, []string{"label1-value1", "new/label2-value"}).addInt64Point(1, 4, 2).addInt64Point(1, 2, 3).
-					addTimeseries(0, []string{"label1-value1", "label2-value4"}).
+				metricBuilder().setName("metric1").setLabels([]string{"label1", "label2"}).
+					setDataType(metricspb.MetricDescriptor_GAUGE_INT64).
+					addTimeseries(1, []string{"label1-value1", "label2-value3"}).
+					addInt64Point(0, 1, 2).
+					addTimeseries(3, []string{"label1-value1", "new/label2-value"}).
+					addInt64Point(1, 4, 2).
+					addTimeseries(3, []string{"label1-value1", "new/label2-value"}).
+					addInt64Point(2, 2, 3).
```

Comment:
Seems like the functionality has changed slightly here. This test is supposed to aggregate values where `label2 = label2-value1` or `label2 = label2-value2` (into `label2 = new/label2-value`). This should be into a single timeseries with 2 data points, but this is now split across 2 timeseries with the same labelset and start timestamp which seems incorrect.

Reply:
In OTLP we don't have timeseries with multiple points we have only metrics with multiple points, so they are equivalent. If this is important we can update the OTLP conversion to OC to re-group similar points into timeseries, but that is extra work on the conversion.